File: Programmers/Gayo/두개_뽑아서_더하기.cpp
#include <string>
#include <vector>
#include <set>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> numbers) {
    set<int> del;

    for(int i=0; i<numbers.size()-1; i++){
        for(int j=i+1; j<numbers.size(); j++){
            del.insert(numbers[i]+numbers[j]);
        }
    }
    vector<int> answer(del.begin(), del.end());
    
    return answer;
}

File: Programmers/Been/Week1/두개_뽑아서_더하기.py
from itertools import combinations
def solution(numbers):
    return sorted(set(x + y for x, y in combinations(numbers, 2)))

def solution(numbers):
    answer = []
    for i in range(len(numbers)):
        for j in range(i + 1, len(numbers)):
            answer.append(numbers[i] + numbers[j])
    return sorted(list(set(answer)))
